src/argument_parse.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import sys
+
+
+def validate_month_year(month_year_str: str) -> tuple:
+    """
+    Description: This function is used for monthly report and generate chart commands to validate the year and month
+
+    Args:
+        month_year_str
+
+    Returns:
+        tuple
+    """
+    try:
+        year, month = map(int, month_year_str.split("/"))
+        if month < 1 or month > 12:
```

Comment:
if the passed month is greater than 12 or less than 0 it won't stop the execution rather it just continue will the invalid values

Reply:
Even if the sys.exit(1) command is there still it will continue ?